spacemesh/v2/node.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package spacemesh.v2;
+
+message NodeStatusRequest {}
+
+message NodeStatusResponse {
+  uint64 connected_peers = 1; // number of connected neighbors
+  bool   is_synced = 2; // true when meshed is synced
+  uint32 synced_layer = 3; // the last layer node has synced
+  uint32 verified_layer = 4; // the last layer node has verified
+  uint32 head_layer = 5; // head layer is the tip
+  uint32 head_block = 6;
```

Comment:
I think we need to add something like `current_layer` which is the current layer number based strictly on time

Reply:
in v1 we have

`synced_layer` - latest layer we saw from the network
`top_layer` - current layer, based on time
`verified_layer` - latest verified layer

what is your proposal?

Reply:
I think we're missing one. Here's my proposal:

```
  uint32 synced_layer = 3; // last layer node has synced
  uint32 verified_layer = 4; // last layer node has verified
  uint32 head_layer = 5; // current chain head; the last layer the node has gossiped or seen
  uint32 current_layer = 6; // current layer, based on clock time
```

`synced_layer` is only useful while syncing. Blocks are also received via gossip. The relationship should be:

synced_layer <= verified_layer <= head_layer <= current_layer